flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -258,7 +258,7 @@ function run_group_2 {
     LOG4J_PROPERTIES=${END_TO_END_DIR}/../tools/ci/log4j.properties
 
     MVN_LOGGING_OPTIONS="-Dlog.dir=${DEBUG_FILES_OUTPUT_DIR} -DlogBackupDir=${DEBUG_FILES_OUTPUT_DIR} -Dlog4j.configurationFile=file://$LOG4J_PROPERTIES"
-    MVN_COMMON_OPTIONS="-Dflink.forkCount=2 -Dfast -Pskip-webui-build"
+    MVN_COMMON_OPTIONS="-Dflink.forkCountITCase=2 -Dflink.forkCountUnitTest=4 -Dfast -Pskip-webui-build"
```

Comment:
I don't think we need to set these explicitly anymore.

Reply:
Ops, good point. Sorry about that.

Reply:
Fixed